sdk/webpubsub/azure-messaging-webpubsub/CHANGELOG.md

```diff
@@ -1,17 +1,17 @@
 # Release History
 
-## 1.4.0-beta.1 (Unreleased)
+## 1.3.4 (2025-03-11)
```

Comment:
Are we not going to release this until March 11th? We might have a newer azure-core by then.